src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -2372,4 +2376,49 @@ protected function paymentTestModeNote($payment)
 
         return $note;
     }
+
+    /**
+     * Method to print the giftcard payment details on debug and order note
+     *
+     * @param Mollie\Api\Resources\Payment $payment
+     * @param WC_Order                     $order
+     *
+     */
+    protected function debugGiftcardDetails(
+            Mollie\Api\Resources\Payment $payment,
+            WC_Order $order
+    ) {
+        $details = $payment->details;
+        if (!$details) {
+            return;
+        }
+        $debugLine = "";
+        foreach ($details->giftcards as $giftcard) {
+            $debugLine .= sprintf(
+                    esc_html_x(
+                            'Mollie - Giftcard details: %1$s %2$s %3$s.',
+                            'Placeholder 1: giftcard issuer, Placeholder 2: amount value, Placeholder 3: currency',
```

Comment:
Context info is just for translators:
![context](https://user-images.githubusercontent.com/456223/88276839-ffd2cd80-ccdf-11ea-904a-0a30843765ac.jpg)

currently the debug line is going to be read like this: `Mollie - Giftcard details: foo bar baz.` so no clue about what these things are other than looking at the code and read this line.